bgw-core/src/main/kotlin/tools/aqua/bgw/builder/UINodeBuilder.kt

```diff
@@ -101,6 +101,9 @@ internal class UINodeBuilder {
 
 			node.textProperty().bindTextProperty(textArea)
 			node.promptText = textArea.prompt
+			textArea.onUpdatePrompt = {
```

Comment:
A ``guiListener`` should be set on an observable property instead of setting a function reference.